_data/nav.yml

```diff
@@ -1,10 +1,34 @@
 - title: "Home"
   href: "/"
 
-- title: "About"
+
+- title: Attend
+  subcategories:
+    - subtitle: Schedule
+      subhref: /schedule
+    - subtitle: Tickets
+      subhref: /tickets
+
+- title: "Sponsors & Friends"
```

Comment:
Can we keep these separate, if possible?

Reply:
Yes, testing this does have enough space to be at the top level